department-of-veterans-affairs/va.gov-cms
# VACMS-3467: CLP Video Panel

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3467

## Testing done


## Screenshots


## QA steps

As user _uid_ with _user_role_

- Login to PR environment
- Navigate here:` /media/add/video`
- Confirm the the following:
  - [x] Video URL *
  - [x] Duration `h:mm:ss`
  - [x] Description `5000 characters limit`
- Complete **Add Video** form and save
  - [x] Save successful
proof: 
![image](https://user-images.githubusercontent.com/5752113/101097144-ac766300-358e-11eb-99a3-a22a2f78c438.png)


- Navigate here: /node/add/campaign_landing_page
- Select the **Video** panel tab
- Confirm the the following:
  - [x] Add a video panel `boolean`
  - [x] Panel Header `70 character limit`
  - [x] Video Entity Reference `Add media`
  - [x] Add a link to more videos `Button` with Button Label and Button Link
- Select Add media button
  - [x] Confirm the new video added from steps above appears and Duration and Description are visible.
- Complete all required fields on other tabs
  - [x] Save successful

- [x] confirm panel toggle follows pattern from stories.


### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
